### DISCUSSION TITLE
Question: Run Nextflow project hosted in CodeCommit

### DISCUSSION THREAD

**Discussion by tamuanand** [nextflow-io/nextflow#2481] [Q&A]
Hi

Is it possible to run Nextflow pipelines hosted in a CodeCommit repo similar to how one can run it with Github/Gitlab - https://nextflow.io/docs/latest/sharing.html?highlight=git#running-a-pipeline

There is [mention from @pditommaso](https://github.com/nextflow-io/nextflow/pull/1594#issuecomment-633266807) here about merging the CodeCommit branch, but it is not clear if the feature is included in a Nextflow release as also asked by @phue in the same thread.

It would be great to include this feature of running Nextflow pipelines using projects hosted in CodeCommit repo.

Thanks in advance.

**Comment by pditommaso** [comment]
This has not yet been solved. There are some not trivial problems with required dependencies to be addressed.

**Reply by tamuanand** [reply]
Thanks @pditommaso  

Based on comments etc made by @wdleepang, it looked as though all requirements were taken care of - however  I could be wrong
